### PR TITLE
fix/V1-048 - added blacklist for tokens

### PR DESCRIPTION
**Pull request by zhuk-vezunchikz**

### CODE REVIEW

File: backend/src/controllers/auth/logout.ts [hunk 1]
```diff
@@ -11,6 +11,12 @@ const logout = async (
     const { id: userId } = res.locals;
     await clearTokenProvider(userId);
     res.clearCookie('refreshToken');
+
+    const accessToken = req.headers.authorization?.split(' ')[1];
```
Comment (dapodev) [inline backend/src/controllers/auth/logout.ts:15]: Let's finally extract this to separate function, for example `extractAccessTokenValue`.

File: backend/src/middlewares/authMiddleware.ts [hunk 2]
```diff
@@ -4,13 +4,19 @@ import isExpectedHttpError from 'utils/typeGuards/isExpectedHttpError';
 import { verifyAccessToken } from 'utils/auth/authUtils';
 import UnauthorizedError from 'classes/errors/clientErrors/UnauthorizedError';
 import ForbiddenError from 'classes/errors/clientErrors/ForbiddenError';
+import { checkTokenInBlacklist } from '../db/providers/authProvider';
```
Comment (dapodev) [inline backend/src/middlewares/authMiddleware.ts:7]: Fix import

**Review by dapodev — CHANGES_REQUESTED** [review]
And please, add creating new collection and removing it to migrations

File: backend/src/controllers/auth/logout.ts (hunk 1 above)
Comment (zhuk-vezunchikz) [reply backend/src/controllers/auth/logout.ts]: done

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**